Description: Upstream changes introduced in version 0.1.0-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 acsilserver (0.1.0-1) unstable; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Emmanuel OHLAND <ohland_e@acsilserver.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getWebPath.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getIsProfilePicture.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setIsProfilePicture.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setName.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setId.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setPath.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$size.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:7:"integer";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$confirm_password.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$username.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:255;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:1;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$salt.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:32;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getFile.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$isProfilePicture.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:7:"integer";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getUploadDate.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getOwner.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setUploadDate.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:2:{i:0;O:26:"Doctrine\\ORM\\Mapping\\Table":5:{s:4:"name";s:4:"User";s:6:"schema";N;s:7:"indexes";N;s:17:"uniqueConstraints";N;s:7:"options";a:0:{}}i:1;O:27:"Doctrine\\ORM\\Mapping\\Entity":2:{s:15:"repositoryClass";s:43:"AcsilServer\\AppBundle\\Entity\\UserRepository";s:8:"readOnly";b:0;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getSize.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$id.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:3:{i:0;O:23:"Doctrine\\ORM\\Mapping\\Id":0:{}i:1;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:7:"integer";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}i:2;O:35:"Doctrine\\ORM\\Mapping\\GeneratedValue":1:{s:8:"strategy";s:4:"AUTO";}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:2:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Entity":2:{s:15:"repositoryClass";N;s:8:"readOnly";b:0;}i:1;O:42:"Doctrine\\ORM\\Mapping\\HasLifecycleCallbacks":0:{}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/7848e01294005dc610fa7d9bc7ca04f657cb55cf.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$pictureAccount.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:255;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getName.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$file.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:44:"Symfony\\Component\\Validator\\Constraints\\File":15:{s:7:"maxSize";s:7:"6000000";s:9:"mimeTypes";a:0:{}s:15:"notFoundMessage";s:28:"The file could not be found.";s:18:"notReadableMessage";s:25:"The file is not readable.";s:14:"maxSizeMessage";s:98:"The file is too large ({{ size }} {{ suffix }}). Allowed maximum size is {{ limit }} {{ suffix }}.";s:16:"mimeTypesMessage";s:86:"The mime type of the file is invalid ({{ type }}). Allowed mime types are {{ types }}.";s:25:"uploadIniSizeErrorMessage";s:72:"The file is too large. Allowed maximum size is {{ limit }} {{ suffix }}.";s:26:"uploadFormSizeErrorMessage";s:22:"The file is too large.";s:25:"uploadPartialErrorMessage";s:37:"The file was only partially uploaded.";s:24:"uploadNoFileErrorMessage";s:21:"No file was uploaded.";s:26:"uploadNoTmpDirErrorMessage";s:46:"No temporary folder was configured in php.ini.";s:27:"uploadCantWriteErrorMessage";s:36:"Cannot write temporary file to disk.";s:27:"uploadExtensionErrorMessage";s:42:"A PHP extension caused the upload to fail.";s:18:"uploadErrorMessage";s:31:"The file could not be uploaded.";s:6:"groups";a:1:{i:0;s:7:"Default";}}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setFile.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$path.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:255;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:1;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$creationDate.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:8:"datetime";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getAbsolutePath.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#upload.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:2:{i:0;O:32:"Doctrine\\ORM\\Mapping\\PostPersist":0:{}i:1;O:31:"Doctrine\\ORM\\Mapping\\PostUpdate":0:{}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$email.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:60;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:1;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$id.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:3:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:7:"integer";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}i:1;O:23:"Doctrine\\ORM\\Mapping\\Id":0:{}i:2;O:35:"Doctrine\\ORM\\Mapping\\GeneratedValue":1:{s:8:"strategy";s:4:"AUTO";}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#preUpload.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:2:{i:0;O:31:"Doctrine\\ORM\\Mapping\\PrePersist":0:{}i:1;O:30:"Doctrine\\ORM\\Mapping\\PreUpdate":0:{}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getId.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#removeUpload.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:31:"Doctrine\\ORM\\Mapping\\PostRemove":0:{}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$owner.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:255;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setSize.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$password.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:40;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$uploadDate.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:8:"datetime";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#setOwner.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c#getPath.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:0:{}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/62dcab9c93a75c9ef86c8022e85b752b20f3290c$name.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:255;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$isActive.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";s:9:"is_active";s:4:"type";s:7:"boolean";s:6:"length";N;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$lastname.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:50;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$roles.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:60;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
--- /dev/null
+++ acsilserver-0.1.0/app/cache/dev/annotations/dd6b6903b285c52fa58c06245801967fc14e256d$firstname.cache.php
@@ -0,0 +1 @@
+<?php return unserialize('a:1:{i:0;O:27:"Doctrine\\ORM\\Mapping\\Column":9:{s:4:"name";N;s:4:"type";s:6:"string";s:6:"length";i:50;s:9:"precision";i:0;s:5:"scale";i:0;s:6:"unique";b:0;s:8:"nullable";b:0;s:7:"options";a:0:{}s:16:"columnDefinition";N;}}');
\ No newline at end of file
